### ISSUE TITLE
Missing files with SLOC

### ISSUE THREAD

**Issue by PatrickAlphaC** [Cyfrin/aderyn#169]
Aderyn is missing files when giving the sloc report.

Steps:

```
git clone https://github.com/Cyfrin/1-horse-store-s23
cd 1-horse-store-s23
aderyn .
```

The report will miss `src/horseStoreV1/HorseStore.sol`. It will instead output:

```
## Files Details

| Filepath | nSLOC |
| --- | --- |
| src/horseStoreV1/HorseStoreYul.sol | 16 |
| src/horseStoreV1/interfaces/ISimpleStorage.sol | 5 |
| src/horseStoreV2/IHorseStore.sol | 7 |
| **Total** | **28** |

```

**Comment by alexroan** [comment]
Oooo great bug, thanks @PatrickAlphaC. This is interesting, and is caused because you have two separate contracts with the same name `HorseStore`, one inside `src/horseStoreV1` and the other inside `src/HorseStorev2`. This is supported by Foundry, but the compiled output folder structure is slightly different in this case.

I'll get on this ASAP!

**Comment by alexroan** [comment]
This fix will be published in the next release 0.0.14 @PatrickAlphaC
